pcsx2/gui/AppCoreThread.cpp

```diff
@@ -510,6 +510,15 @@ void AppCoreThread::ApplySettings( const Pcsx2Config& src )
 	{
 		_parent::ApplySettings( fixup );
 	}
+
+	if (m_ExecMode >= ExecMode_Paused)
+	{
+		// Disable Vsync when frame limiter is disabled
```

Comment:
It makes more sense to place the comment near the else statement (or) change the check into ``if (!g_Conf->EmuOptions.GS.FrameLimitEnable)`` but I think I'm being too nitpicky here, your choice.